### ISSUE TITLE
[2.8] [WebProfilerBundle] Multiple tabs in the same page do not work correctly

### ISSUE THREAD

**Issue by csarrazi** [symfony/symfony#16506]
It seems that embedding the tab control multiple times breaks the tab feature:

At first, it seems okay.

<img width="1083" alt="step1" src="https://cloud.githubusercontent.com/assets/465798/11036315/30b1a7b0-86f8-11e5-97eb-0e475862393e.png">

But if you click on the second tab of the first tab group:

<img width="1123" alt="step2" src="https://cloud.githubusercontent.com/assets/465798/11036365/747b61fc-86f8-11e5-86c4-9f64bc43f104.png">

And then, if you try clicking on the first tab of the last group:

<img width="1080" alt="step3" src="https://cloud.githubusercontent.com/assets/465798/11036370/7b278c38-86f8-11e5-8238-2f7c9eb71d85.png">

The code used is the following:

```twig
{% block panel %}
    <div class="sf-tabs">
        <div class="tab">
            <h3 class="tab-title">Request</h3>
            <div class="tab-content">
                toto
            </div>
        </div>
            <div class="tab">
                <h3 class="tab-title">Response</h3>
                <div class="tab-content">
                    titi
                </div>
            </div>
    </div>
{% endblock %}
```

**Comment by csarrazi** [comment]
Ping @javiereguiluz

**Comment by javiereguiluz** [comment]
@csarrazi it should be fixed as of #16515

**Comment by csarrazi** [comment]
Yes indeed! :)
